Assets/Scripts/Models/Buildable/Utility.cs

```diff
@@ -549,6 +551,15 @@ public void Deconstruct(Utility utility)
             Removed(this);
         }
 
+        if (deconstructInventory != null)
+        {
+            foreach (Inventory inv in deconstructInventory)
+            {
+                inv.MaxStackSize = PrototypeManager.Inventory.Get(inv.Type).maxStackSize;
```

Comment:
Right, I forgot that we shouldn't relay on the prototype loading order. Maybe we shouldn't even create other entities while creating a prototype. An option would be to save just the type and the amount when reading the xml and create the inventory at this point.